pkg/controller/storagecluster/initialization_reconciler_test.go

```diff
@@ -191,6 +198,14 @@ func assertExpectedResources(t assert.TestingT, reconciler ReconcileStorageClust
 
 	expected, err := reconciler.newStorageClasses(cr)
 	assert.NoError(t, err)
+	if isValidCloudPlatform(reconciler.platform.platform) {
+		assert.Equal(t, len(expected), 2)
+	} else {
+		assert.Equal(t, len(expected), 3)
+		request.Name = "ocsinit-ceph-Delete-bucket"
+		err = reconciler.client.Get(nil, request.NamespacedName, actualSc3)
+		assert.NoError(t, err)
```

Comment:
Test more than just thing. Do the ownerref and field checks the other StorageClasses get.

Reply:
Done...